src/pages/[platform]/build-a-backend/existing-resources/cdk/index.mdx

```diff
@@ -0,0 +1,865 @@
+    safePrint('Error configuring Amplify: $e');
+  }
+}
+
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  // GoRouter configuration
+  static final _router = GoRouter(
+    routes: [
+      GoRoute(
+        path: '/',
+        builder: (context, state) => const HomePage(),
+      ),
+      GoRoute(
+        path: '/manage-todo-item',
+        name: 'manage',
+        builder: (context, state) => ToDoItemPage(
+          todoItem: state.extra as Todo?,
+        ),
+      ),
+    ],
+  );
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp.router(
+      routerConfig: _router,
+      debugShowCheckedModeBanner: false,
+      builder: (context, child) {
+        return child!;
+      },
+    );
+  }
+}
+
+```
+
+**Step 12:** Run the app in the Chrome browser using the following command.
+
+```dart showLineNumbers={false}
+
+flutter run -d chrome 
+
+```
+
+<video autoPlay={true} muted={true} loop={true} width="100%">
+  <source src="/images/existing-resources/todo-example-cdk.mp4" />
+</video>
+
+Congratulations on successfully creating a new Flutter app and linking it to a preexisting GraphQL API built with AWS CDK. The next optional section will guide you through doing the same for a React web app. Alternatively, you can skip ahead to the ["Clean up resources"](/[platform]/build-a-backend/existing-resources/cdk/#clean-up-resources) section.
+
+## Build a React app and connect to the GraphQL API
+
+In this section, we will connect a React web app to our existing GraphQL API. First, we will create a new React project and install the necessary Amplify packages. Next, we will use the Amplify CLI to generate GraphQL code matching our API structure. Then, we will add React components to perform queries and mutations to manage to-do items in our API. After that, we will configure the Amplify library with details of our backend API. Finally, we will run the application to demonstrate full CRUD functionality with our existing API. 
+
+**Step 1:** Create a React app by running the following command in your terminal.
+
+```react showLineNumbers={false}
+
+npx create-react-app react-amplify-connect 
+
+```
+
+**Step 2:** Open the newly created React app using VS Code, or your preferred IDE.
+ 
+**Step 3:** Install the aws-amplify, @aws-amplify/ui-react, and @aws-amplify/cli packages by running the following commands. 
+
+```react showLineNumbers={false}
+
+npm install aws-amplify @aws-amplify/ui-react @aws-amplify/cli 
+
+```
+
+**Step 4:** Use the awsAppsyncApiId and awsAppsyncRegion values of the CDK stack you created previously to generate the GraphQL client helper code by running the following command.
+
+<img src="/images/existing-resources/outputs-cdk.png" alt="awsAppsyncApiId and awsAppsyncRegion values highlighted within the outputs of the CDK stack." width="750" />
+
+```react
+
+npx @aws-amplify/cli codegen add --apiId <awsAppsyncApiId> --region <awsAppsyncRegion> 
+
+```
```

Comment:
```suggestion
```react
npx @aws-amplify/cli codegen add --apiId <aws-appsync-api-id> --region <aws-appsync-region> 
```
```

revised casing of placeholder text